test/srt/test_json_constrained.py

```diff
@@ -121,5 +121,114 @@ def test_mix_json_and_other(self):
             list(executor.map(self.run_decode, json_schemas))
 
 
+class TestJSONConstrainedXGrammarBackend(unittest.TestCase):
```

Comment:
Do not copy the code. You can inherit the above class and only rewrite the setUpClass function.